kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+
+  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
```

Comment:
@whitzhu Awesome! I saw that the order of components here is different that on master, just making sure that it is a conscious decision and that it mirrors the visual order of the elements in the page. :wink: